docker-compose.av.yml

```diff
@@ -145,12 +134,13 @@ services:
     user: "2000:2000"
   ds-cv-inference:
     image: "automated-vending/ds-cv-inference:dev"
-    container_name: automated-checkout_ds-cv-inference
+    container_name: automated-vending_ds-cv-inference
     command: ["/go/src/ds-cv-inference/images","mqtt-broker:1883","0.85","/go/src/ds-cv-inference/skumapping.json"]
     depends_on:
       mqtt-broker:
         condition: service_started
     environment:
+      EDGEX_SECURITY_SECRET_STORE: "false"
       SERVICE_HOST: ds-cv-inference
```

Comment:
This is not an EdgeX based service so these emv can also be removed.

Reply:
Just the EDGEX_SECURITY_SECRET_STORE or the SERVICE_HOST also?

Reply:
Both